qa/L0_string_io/string_client_test.py

```diff
@@ -92,12 +90,36 @@ def test_tf_unicode_bytes(self):
 
         results = triton_client.infer(model_name=model_name,
                                       inputs=inputs,
-                                      outputs=outputs,
-                                      model_version=model_version)
+                                      outputs=outputs)
 
+        out0 = results.as_numpy('OUTPUT0')
         # We expect there to be 1 results (with batch-size 1). Verify
         # that all 8 result elements are the same as the input.
-        self.assertTrue(np.array_equal(in0, results.as_numpy('OUTPUT0')))
+        self.assertTrue(np.array_equal(in0, out0))
+
+        in0_object = in0.astype(np.object)
```

Comment:
So this is testing that np.object works for both strings and binary blobs?
Does np.bytes_ work for both cases as well? Can you test that?

Reply:
`|S78` is like `np.bytes_`. I'll add a test case to be more explicit.